app/exporters/formatters/asylum_support_decision_indexable_formatter.rb

```diff
@@ -0,0 +1,23 @@
+require "formatters/abstract_specialist_document_indexable_formatter"
+
+class AsylumSupportDecisionIndexableFormatter < AbstractSpecialistDocumentIndexableFormatter
+  def type
+    "asylum_support_decision"
+  end
+
+private
+  def extra_attributes
+    {
+      tribunal_decision_decision_date: entity.tribunal_decision_decision_date,
+      tribunal_decision_judges: entity.tribunal_decision_judges,
```

Comment:
@tijmenb Is it ok with you if we merge what we have so far, and make the changes you have suggested in a new branch?